packages/react-components/react-link/library/src/contexts/inlineLinkContext.ts

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+
+export type InlineLinkContextValue = {
+  inlineLinks: boolean;
+};
+
+const inlineLinkContext = React.createContext<InlineLinkContextValue | undefined>(undefined);
+
+export const inlineLinkContextDefaultValue: InlineLinkContextValue = {
+  inlineLinks: false,
+};
+
+export const InlineLinkContextProvider = inlineLinkContext.Provider;
+export const useInlineLinkContext = () => React.useContext(inlineLinkContext) ?? inlineLinkContextDefaultValue;
```

Comment:
```suggestion
export type LinkContextValue = {
  inline?: boolean;
};

const LinkContext = React.createContext<LinkContextValue | undefined>(undefined);

export const linkContextDefaultValue: LinkContextValue = {
  inlineLinks: false,
};

export const LinkContextProvider = LinkContext.Provider;
export const useLinkContext = () => React.useContext(LinkContext) ?? linkContextDefaultValue;
```

Let's make it generic like `AvatarContext`.